webpack/assets/javascripts/react_app/components/common/AutoComplete/index.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import Downshift from 'downshift';
+import classnames from 'classnames';
+import { InputGroup } from 'react-bootstrap';
```

Comment:
You should be able to import it from `patternfly-react`
https://rawgit.com/patternfly/patternfly-react/gh-pages/index.html?knob-Disabled=false&selectedKind=Forms&selectedStory=Input%20Groups&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs